client/src/components/common/Checkbox/index.css

```diff
@@ -0,0 +1,36 @@
+.checkbox__home{
+  display: inline-flex;
+  margin: 10px;
+}
+
+.checkbox__input[type="checkbox"] {
```

Comment:
you shouldn't need the attribute selector, the class uniquely identifies the element and the type doesn't change